app/client/components/Layout.tsx

```diff
@@ -1,12 +1,30 @@
-import React, { PropsWithChildren, useContext } from "react";
-
 import { Context } from "@reactivated";
+import React, { PropsWithChildren, useContext, useEffect } from "react";
 import { Helmet } from "react-helmet-async";
+import toast, { Toaster } from "react-hot-toast";
+import { CustomToast } from "./CustomToast";
 import { Footer } from "./Footer";
 import { Navbar } from "./navbar/Navbar";
 
 export const Layout = ({ children }: PropsWithChildren) => {
   const context = useContext(Context);
+  const messages = context.messages;
```

Comment:
You could use object destructing: `const { message } = useContext(context);`

Reply:
Done